### ISSUE TITLE
About number of parameters

### ISSUE THREAD

**Issue by gaogg** [ivclab/CPG#3]
Thanks for your work and code.
I‘m a bit confusing after reading the code. I'd like to know whether the mask and piggymask are different thing, the former works for pruning and the latter works for picking weights for fine-tuning, and they both have to be saved for inference, right?

**Comment by HaoranZhuExplorer** [comment]
Yes, I think the mask and the piggymask are two things. The mask is used for getting the network for previous tasks; the piggymask is used for picking useful neurons for the current task from previous tasks by using the piggyback approach.
